05week/checkers.js

```diff
@@ -52,15 +61,57 @@ function Board() {
     console.log(string);
   };
 
+  this.createCheckers = function() {
+    // // [row, col]
+    const whitePos = [
+     [0, 1], [0, 3], [0, 5], [0, 7],
+     [1, 0], [1, 2], [1, 4], [1, 6],
+     [2, 1], [2, 3], [2, 5], [2, 7],
+    ]
+    for (let i = 0; i < 12; i++) {
+     let whiteRow = whitePos[i][0];
+     let whiteCol = whitePos[i][1];
+     let whiteChecker = new Checker('white');
+     this.checkers.push(whiteChecker);
+     this.grid[whiteRow][whiteCol] = whiteChecker;
+    }
+
+
+    const blackPos = [
+     [5, 0], [5, 2], [5, 4], [5, 6],
+     [6, 1], [6, 3], [6, 5], [6, 7],
+     [7, 0], [7, 2], [7, 4], [7, 6],
+    ]
+    for (let i = 0; i < 12; i++) {
+     let blackRow = blackPos[i][0];
+     let blackCol = blackPos[i][1];
+     let blackChecker = new Checker('black');
+     this.checkers.push(blackChecker);
+     this.grid[blackRow][blackCol] = blackChecker;
+    }
+  };
   // Your code here
+  this.selectChecker(row, column) {
```

Comment:
The `function()` keyword should used in all your methods when handling arguments.